### PR TITLE
chore: added  Educational Website

### PR DESCRIPTION
**Pull request by Ks103**
## Related Issue
#1442 

## Description

This is the educational website, one can practice their skills and there are many resources also available which can help to crack GATE.
  

## Screenshots

Screenshot (209)

## Checklist for projects addition

 

- [x] My project has contribution guidelines.
- [x] My project has a Code of Conduct.
- [x] My project has a README file.
- [x] My project has a License.

---

Please note that if your project doesn't meet these requirements, your PR may be closed. Make sure to include these elements in your project before submitting the PR.

### CODE REVIEW

**Review by priyankarpal — APPROVED** [review]

File: src/DB/projects.json [hunk 1]
```diff
@@ -1268,6 +1268,38 @@
...
+  "Social_media": {
+    "gitHub": "https://github.com/Ks103",
+    "LinkedIn": "https://www.linkedin.com/in/khushi-shroff-70b049211/",
+    "Instagram": "",
+    "YouTube": "",
+    "Twitter": ""
+  },
+  "Projects": [
+    {
+      "link": "https://github-usernamefind.netlify.app/",
```
Comment (priyankarpal) [inline src/DB/projects.json:1282]: add github repo link

File: src/DB/projects.json [hunk 2]
```diff
@@ -1268,6 +1268,38 @@
...
+      "title": "Github-user-info",
+      "description": "This website is of finding username on github so that everyone can access their prs, issues,repositories.",
+        "tech": ["Javascript", "Css", "Html", "React","Tailwind CSS"]
+
+        
+    },
+    {
+      "link": "https://khushitech.netlify.app/",
```
Comment (priyankarpal) [inline src/DB/projects.json:1290]: add github repo link

File: src/DB/projects.json [hunk 3]
```diff
@@ -1268,6 +1268,38 @@
...
+      "title": "Portfolio site",
+      "description": "This is my personal portfolio website connect with me. Let's make this contribution journey more far by connecting with each other. #Happy Learning",
+      "tech": "[Html,Css,Js]"
+    },
+    {
+      "link": "https://communicore.netlify.app/",
```
Comment (priyankarpal) [inline src/DB/projects.json:1296]: add github repo link

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by Ks103** [comment]
What changes are there @priyankarpal ? I have fixed please check

**Comment by priyankarpal** [comment]
> What changes are there @priyankarpal ? I have fixed please check

you have added production links but you need to add github repo link. After I can merge

**Comment by Ks103** [comment]
done with the changes @priyankarpal

File: src/DB/projects.json [hunk 4]
```diff
@@ -1268,6 +1247,38 @@
...
+      "description": "This website is of finding username on github so that everyone can access their prs, issues,repositories.",
+        "tech": ["Javascript", "Css", "Html", "React","Tailwind CSS"]
+
+        
+    },
+    {
+      "link": "https://github.com/Ks103/PortFolio",
+      "title": "Portfolio site",
+      "description": "This is my personal portfolio website connect with me. Let's make this contribution journey more far by connecting with each other. #Happy Learning",
+      "tech": "[Html,Css,Js]"
```
Comment (priyankarpal) [inline src/DB/projects.json:1272]: change these tags 
```suggestion
      "tech": ["Html","Css","Js"]
```

File: src/DB/projects.json [hunk 5]
```diff
@@ -1268,6 +1247,38 @@
...
+    },
+    {
+      "link": "https://github.com/Ks103/CommuniCore",
+      "title": "Educational Website",
+      "description": "This is the educational website, one can practice their skills and there are many resources also available which can help to crack GATE.",
+      "tech": "[Html,Css,Js]"
```
Comment (priyankarpal) [inline src/DB/projects.json:1278]: change these tags
```suggestion
      "tech": ["Html","Css","Js"]
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by Ks103** [comment]
Done with the changes @priyankarpal

**Review by priyankarpal — APPROVED** [review]

**Pull request merged**